index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="style.css">
+	<meta charset="UTF-8">
+	<title>Microverse trials</title>
+</head>
+<body>
+<main id="main">
+	<div id="title-container">
+		<h1 id="title"> Lionel Andres Messi Cuccittini </h1> 
+		<h3> The new king after Maradona </h3>
+	</div>
+	<div id= "img-div"> 
+		<img id="image" src="./messi.jpeg" alt=""> 
+		<p id="img-caption">Messi winning the world cup 2022</p>
+	</div>
+	<div id="tribute-info">
+		<ul>
+			<li><strong>1987</strong>- Born in Rosario, Argentina</li>
+			<li><strong>2000</strong>- Signed with Barcelona and Moved out to Spain</li>
+			<li><strong>2004</strong>- Debut with Barcelona FC</li>
+			<li><strong>2008</strong>- He won the "Triplete" with Barcelona</li>
+			<li><strong>2009</strong>- He won his first Ballon D'Or</li>
+			<li><strong>2012</strong>- He broke the world record Scoring 92 goals in a Calendar year</li>
+			<li><strong>2014</strong>- He reached the World Cup final and lost against Germany</li>
+			<li><strong>2015</strong>- FC Barcelona Signed Luis Suarez and Neymar Jr and became the greatest strikers in the World "the MSN"</li>
+			<li><strong>2019</strong>- He wons his sixth Ballon D'or </li>
+			<li><strong>2021</strong>- He finished his FC Barcelona's contract after 17 years and 35 titles, He won his seventh Ballon D'or and Conquered the Copa America againts Brazil</li>
+			<li><strong>2022</strong>- He won the "Finalissima" Against Italy and the World cup championship against France</li>
+		</ul>
+	</div>
+	<div id="tribute-link"><a href="https://es.wikipedia.org/wiki/Lionel_Messi" target="_blank">If you have time, you should read more about this Fantastic Player</a></div>
+
```

Comment:
Please consider using section tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. The container of the section should be a section tag 🙏🏼